content/en/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Dual-stack support with kubeadm
+feature:
+  title: Dual-stack support with kubeadm
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+
+IPv4/IPv6 dual-stack enables the allocation of both IPv4 and IPv6 addresses to {{< glossary_tooltip text="Pods" term_id="pod" >}} and {{< glossary_tooltip text="Services" term_id="service" >}}.
+
+<!-- body -->
+
+## Enable dual-stack via kubeadm
+
+Install kubeadm following the steps from the [Installing Kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) documentation.
+
+Make sure that nodes allow IPv6 forwarding, if not, run `sudo sysctl -w net.ipv6.conf.all.forwarding=1` on every node in the cluster.
+
+{{< note >}}
+`kubeadm upgrade` will change `IPv6DualStack` to true by default if the feature gate is not set in an old cluster. However, cluster CIDR and service CIDR modifications are not supported.
+{{< /note >}}
+
+### Create a dual-stack cluster
+
+To create a dual-stack cluster with `kubeadm init` you can pass the following flags:
+
+```shell
+kubeadm init --pod-network-cidr=10.244.0.0/16,fd00:10:244::/56 --service-cidr=10.96.0.0/16,fd00:10:96::/112
+```
+
+To make things clearer, here is an example kubeadm [configuration file](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2) `kubeadm-config.yaml` for the primary dual-stack control plane node.
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+featureGates:
+  IPv6DualStack: true
+networking:
+  podSubnet: 10.244.0.0/16,fd00:10:244::/56
+  serviceSubnet: 10.96.0.0/16,fd00:10:96::/112
```

Comment:
```suggestion
  podSubnet: 10.244.0.0/16,2001:db8:42:0::/56     # example ranges
  serviceSubnet: 10.96.0.0/16,2001:db8:42:1::/112 # example ranges
```

Reply:
hehe, that is my fault, those are the default kind valudes, Tim is always spot on these things, docs recommended ranges are 2001:db8